src/utils/components/InstanceTypeModal/ConnectedInstanceTypeModal.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { FC } from 'react';
+
+import useInstanceTypesAndPreferences from '@catalog/CreateFromInstanceTypes/state/hooks/useInstanceTypesAndPreferences';
+
+import ErrorAlert from '../ErrorAlert/ErrorAlert';
+import Loading from '../Loading/Loading';
+
+import InstanceTypeModal, { InstanceTypeModalProps } from './InstanceTypeModal';
+
+const ConnectedInstanceTypeModal: FC<
```

Comment:
The name of this compent isnt clear, what does it mean connected? Please try rename to something clearer

Reply:
The naming comes from  React Redux - in this context it's just that (some) data is connected to the component.
We can change it - what do you propose?

Reply:
I will call this InstanceTypeModal and the inner component InstanceTypeModalContent or some variation

Reply:
In the end I've used `StandaloneInstanceTypeModal`. The `InstanceTypeModalContent` was not accurate because both components are used directly.